src/components/table/renderer.tsx

```diff
@@ -14,7 +15,7 @@ const MAX_LENGTH = 150;
 export default class Table extends React.PureComponent<Props> {
   public render() {
     const headerNodes = this.props.header.map(h => (
-      <th title={h} key={h}>
+      <th onClick={() => this.props.onClickHandler && this.props.onClickHandler(h)} title={h} key={h}>
```

Comment:
When would the handler not be defined?

Reply:
When we want to reuse table without the debugger.

Reply:
Okay